This patch is to make libomxil Makefile support "make -jN".
The omxregister_bellagio stuffs depend on libomxil_bellagio library.

Upstream-Status: Pending

Signed-off-by: Shane Wang <shane.wang@intel.com>

diff -r f59d077d3dd5 Makefile.am
--- a/src/Makefile.am	Wed Dec 28 10:54:36 2011 +0800
+++ b/src/Makefile.am	Wed Dec 28 10:55:46 2011 +0800
@@ -7,6 +7,7 @@
 omxregister_bellagio_SOURCES = omxregister.c common.c common.h
 omxregister_bellagio_CFLAGS = -DOMXILCOMPONENTSPATH=\"$(plugindir)/\" \
 			      -I$(top_srcdir)/include
+omxregister_bellagio_LDADD = $(lib_LTLIBRARIES)
 omxregister_bellagio_LDFLAGS = -lomxil-bellagio -L$(builddir)
 
 lib_LTLIBRARIES = libomxil-bellagio.la
